### DISCUSSION TITLE
any relevant paper?

### DISCUSSION THREAD

**Discussion by dragen1860** [snakers4/silero-vad#107] [Q&A]
Dear all:
  Thank you for your contribution to the whole voice community. We really appreciate the method and pretrained models.
  I wonder is there any related paper or document to illustrate inner details of your work? specifically, i would like to know what kind of network architecture you used? what dataset you used? Thank you again.

**Comment by snakers4** [comment]
Hi,

Since we are not an academic institution and / or corporate-backed entity, publishing papers is not among our top priorities.

Though we publish a lot of down-to-earth articles and even some academic ones - https://github.com/snakers4/silero-models#further-reading.

Since we are for-profit self-financed company, it is not in our best interest to fully open our pipelines.

> I wonder is there any related paper or document to illustrate inner details of your work? 

Only README in this repo + discussions / tickets are public now.

> specifically, i would like to know what kind of network architecture you used? what dataset you used? 

All of these were developed in-house, in many aspects derivative of our STT systems.
